src/stores/events/EventsDataStore.ts

```diff
@@ -284,14 +292,34 @@ export default class EventsDataStore {
 					if (cachedRootNode) {
 						rootNode = cachedRootNode;
 					} else {
-						rootNode = getErrorEventTreeNode(rootNode?.parentId || parentId);
+						rootNode = unknownRoot;
+						if (currentParentId) {
```

Comment:
I wonder if our understanding of this task is different. Suppose we have 3 events, 2 of them are unknown
I expect tree to look like this:
-- event1
-- unknownRoot
   ---- event2
        ---- event2 children
   ---- event3
        ---- event3 children

but you set all the children of unknown event to unknownRoot

that's why i said we need another not root unknown event